update jetbrains toolbox to latest version

arielhernandezmusa opened the pull request:
## Description of changes

<!--
For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [x] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

9ary committed:
nixos/pam/u2f: implement RFC0042

This module has a lot of options, so it's a good candidate for freeform
settings.

wahjava committed:
nfs-utils: enable ldap support

wahjava committed:
nfs-utils: 2.6.2 -> 2.6.4

heywoodlh committed:
choose-gui: init 1.3.1

Release: https://github.com/chipsenkbeil/choose/releases/tag/1.3.1

onny committed:
eintopf: init 0.13.16

redyf committed:
tmuxPlugins.tokyo-night-tmux: init at 1.5.3

Commit:
rtl8821ce: unstable-2024-01-20 -> unstable-2024-03-26

Commit:
rtl8821ce: add defelo to maintainers

bandithedoge committed:
lutris: 0.5.16 -> 0.5.17

https://github.com/lutris/lutris/releases/tag/v0.5.17

Guanran928 committed:
dopamine: init at 3.0.0-preview.26

Co-authored-by: Leonardo Eugênio <lelgenio@lelgenio.com>
Co-authored-by: Sandro <sandro.jaeckel@gmail.com>
Co-authored-by: Aleksana <alexander.huang.y@gmail.com>

fgaz committed:
astromenace: 1.4.1 -> 1.4.2

fgaz committed:
astromenace: add fgaz to maintainers

tobim committed:
clang_18: Upstream patch to fix an ICE

See https://github.com/llvm/llvm-project/pull/93206.

Co-authored-by: Alyssa Ross <hi@alyssa.is>

azuwis committed:
moonlight-qt: Use upstream patches to handle darwin prebuilts

azuwis committed:
moonlight-qt: New upstream deps libplacebo and vulkan-headers

zmitchell committed:
moonlight-qt: 5.0.1 -> 6.0.0

azuwis committed:
moonlight-qt: Move to pkgs/by-name

azuwis committed:
moonlight-qt: Format using nixfmt-rfc-style and sort args

heywoodlh committed:
maintainers: add heywoodlh

Commit:
envision: init at 0-unstable-2024-06-25

Mynacol committed:
nixos/nextcloud: Remove --preserve-env in sudo

This helps supporting sudo-rs, which currently does not implement the
--preserve-env flag and probably won't so in the foreseeable future [1].

The replacement just sets both environment variables behind the sudo
invocation with env, as sudo-rs also doesn't implement env var lists.

The OC_PASS variable is dropped, as it is seemingly unused and would
leak through this approach through /proc.

[1] https://github.com/memorysafety/sudo-rs/issues/129

jpds committed:
nixos/graylog: Added more test gates

jpds committed:
nixos/graylog: Increased disk size, fixes preflight check:

Preflight check failed with error: Journal directory
</var/lib/graylog/data/journal> has not enough free space (3412 MB) available.
You need to provide additional 1707 MB to contain 'message_journal_max_size = 5120 MB'

GaetanLepage committed:
ruff: 0.4.10 -> 0.5.0

Diff: https://github.com/astral-sh/ruff/compare/refs/tags/0.4.10...0.5.0

Changelog: https://github.com/astral-sh/ruff/releases/tag/0.5.0

GaetanLepage committed:
ruff-lsp: 0.0.53 -> 0.0.54

Diff: https://github.com/astral-sh/ruff-lsp/compare/refs/tags/v0.0.53...v0.0.54

Changelog: https://github.com/astral-sh/ruff-lsp/releases/tag/v0.0.54

vbgl committed:
framac: 28.1 (Nickel) → 29.0 (Copper)

r-ryantm committed:
prometheus-knot-exporter: 3.3.6 -> 3.3.7

kilimnik committed:
gitlab-release-cli: init at 0.18.0

Scrumplex committed:
sops: add update script

Signed-off-by: Sefa Eyeoglu <contact@scrumplex.net>

Scrumplex committed:
sops: 3.8.1 -> 3.9.0

Signed-off-by: Sefa Eyeoglu <contact@scrumplex.net>

infinisil committed:
workflows/check-by-name: link to githubstatus

Co-authored-by: Philip Taron <philip.taron@gmail.com>

Tochiaha committed:
python312Packages.huepy: init at 1.2.1

pyrox0 committed:
undollar: remove from nodePackages

Commit:
halloy: build .app on darwin

r-ryantm committed:
lombok: 1.18.32 -> 1.18.34

Commit:
hyprls: init at 0.2.0

Commit:
halloy: add myself as maintainer

azuwis committed:
moonlight-qt: Add passthru.updateScript

azuwis committed:
moonlight-qt: 6.0.0 -> 6.0.1

azuwis committed:
moonlight-qt: Add myself as maintainer

OPNA2608 committed:
tests/lomiri: Fix OCR detection

Scrumplex committed:
sops: update homepage

Signed-off-by: Sefa Eyeoglu <contact@scrumplex.net>

romildo committed:
marwaita-mint: init at 20.2-unstable-2024-07-01

romildo committed:
marwaita: move to pkgs/by-name/ma/

romildo committed:
marwaita: 20.1 -> 20.2-unstable-2024-07-01

Diff: https://github.com/darkomarko42/marwaita/compare/20.1...20.2

romildo committed:
marwaita-peppermint: move to pkgs/by-name/ma/

r-ryantm committed:
uv: 0.2.15 -> 0.2.21

Diff: https://github.com/astral-sh/uv/compare/0.2.15...0.2.21

Changelog: https://github.com/astral-sh/uv/releases/tag/0.2.21

Commit:
nixos/envision: init module

Commit:
nixos/doc/rl-2411: Document programs.envision

romildo committed:
marwaita-manjaro: move to pkgs/by-name/ma/

romildo committed:
marwaita-manjaro: rename to marwaita-darkcyan

romildo committed:
marwaita-darkcyan: add update script

romildo committed:
marwaita-darkcyan: 20.1 -> 20.2-unstable-2024-07-01

Diff: https://github.com/darkomarko42/marwaita-darkcyan/compare/20.1...ad837492a377909e0e84f090c740f0296d311797

romildo committed:
marwaita-peppermint: rename to marwaita-red

romildo committed:
marwaita-red: 17.0 -> 20.2-unstable-2024-07-01

thomasjm committed:
julia.withPackages: fix for overrides not in a registry

hacklschorsch committed:
preCloneScript is now preGetSourcesScript

The --pre-clone-script service configuration has been deprecated (https://docs.gitlab.com/ee/update/deprecations.html#deprecation-and-planned-removal-for-ci_pre_clone_script-variable-on-gitlab-saas) and replaced by --pre-get-sources-script (https://about.gitlab.com/blog/2023/03/27/changes-to-the-preclonescript/).

bobby285271 committed:
epiphany: 46.1 → 46.2

https://gitlab.gnome.org/GNOME/epiphany/-/compare/46.1...46.2

bobby285271 committed:
evolution: 3.52.2 → 3.52.3

https://gitlab.gnome.org/GNOME/evolution/-/compare/3.52.2...3.52.3

bobby285271 committed:
evolution-data-server: 3.52.2 → 3.52.3

https://gitlab.gnome.org/GNOME/evolution-data-server/-/compare/3.52.2...3.52.3

ENABLE_UOA is gone for a while:
https://gitlab.gnome.org/GNOME/evolution-data-server/-/commit/66121edf92f8babe8de8d970faf2155ca8824c07

bobby285271 committed:
evolution-ews: 3.52.2 → 3.52.3

https://gitlab.gnome.org/GNOME/evolution-ews/-/compare/3.52.2...3.52.3

bobby285271 committed:
gexiv2: 0.14.2 → 0.14.3

https://gitlab.gnome.org/GNOME/gexiv2/-/compare/gexiv2-0.14.2...gexiv2-0.14.3

bobby285271 committed:
gnome.gnome-control-center: 46.2 → 46.3

https://gitlab.gnome.org/GNOME/gnome-control-center/-/compare/46.2...46.3

Some tests are now disabled if Xvfb is missing, I imagine we don't need xvfb-run when
the X11SessionTestCase helper actually takes care of X and system/session bus.
https://gitlab.gnome.org/GNOME/gnome-control-center/-/commit/51bf36ac68051aaf183c1aa0b45b8af7cc479c70

bobby285271 committed:
gnome.gnome-initial-setup: 46.2 → 46.3

https://gitlab.gnome.org/GNOME/gnome-initial-setup/-/compare/46.2...46.3

I don't know why we need to set PKG_CONFIG_SYSTEMD_SYSUSERSDIR so I just dropped it,
it looks like the prefix is properly replaced in upstream's meson.build via pkgconfig_define.

bobby285271 committed:
gnome-online-accounts: 3.50.2 → 3.50.3

https://gitlab.gnome.org/GNOME/gnome-online-accounts/-/compare/3.50.2...3.50.3

bobby285271 committed:
orca: 46.1 → 46.2

https://gitlab.gnome.org/GNOME/orca/-/compare/ORCA_46_1...ORCA_46_2

bobby285271 committed:
gnome.gnome-remote-desktop: 46.2 → 46.3

https://gitlab.gnome.org/GNOME/gnome-remote-desktop/-/compare/46.2...46.3

bobby285271 committed:
gnome.gnome-shell: 46.2 → 46.3.1

https://gitlab.gnome.org/GNOME/gnome-shell/-/compare/46.2...46.3.1

I am mostly confused by b9b5f9a, actually I don't think we need jasmine-gjs in 46.x.

Dropping gtk3 and libstartup_notification:
https://gitlab.gnome.org/GNOME/gnome-shell/-/commit/27b9985da3f4e098e1bb9aa27a15d9151af7efc4

Verified that gtk3 is really not needed by `grep -nr '3\.0'` in src.

While at it explicitly add libX11 and libxml2 as direct deps.

bobby285271 committed:
gnome.gnome-software: 46.2 → 46.3

https://gitlab.gnome.org/GNOME/gnome-software/-/compare/46.2...46.3

bobby285271 committed:
gnome.gnome-sudoku: 46.2 → 46.3

https://gitlab.gnome.org/GNOME/gnome-sudoku/-/compare/46.2...46.3

bobby285271 committed:
libpeas2: 2.0.2 → 2.0.3

https://gitlab.gnome.org/GNOME/libpeas/-/compare/2.0.2...2.0.3

bobby285271 committed:
gnome.mutter: 46.2 → 46.3

https://gitlab.gnome.org/GNOME/mutter/-/compare/46.2...46.3

hacklschorsch committed:
Gitlab-Runner: Add `--post-get-sources-script`

`--post-get-sources-script` has been added a while ago.  This
makes it available via the nix configuration.

See https://about.gitlab.com/blog/2023/03/27/changes-to-the-preclonescript/

Limosine committed:
vnote: 3.17.0 -> 3.18.0

onny committed:
nixos/eintopf: init

pbsds committed:
dolphin-emu: 5.0-21460 -> 2407

Diff: https://github.com/dolphin-emu/dolphin/compare/a9544510468740b77cf06ef28daaa65fe247fd32...b92e354389bb7c0bd114a8631b8af110d3cb3a14

Announcement: https://dolphin-emu.org/blog/2024/07/02/dolphin-releases-announcement/

pbsds committed:
dolphin-emu: add maintainer pbsds

r-ryantm committed:
chezmoi: 2.49.1 -> 2.50.0

r-ryantm committed:
trealla: 2.53.2 -> 2.53.10

onny committed:
nixos/tests/eintopf: init

dotlambda committed:
bitwarden-desktop: 2024.6.0 -> 2024.6.4

Diff: https://github.com/bitwarden/clients/compare/desktop-v2024.6.0...desktop-v2024.6.4

Changelog: https://github.com/bitwarden/clients/releases/tag/desktop-v2024.6.4

romildo committed:
latexminted: 0.1.0b5 -> 0.1.0b9

znaniye committed:
odin: dev-2024-06 -> dev-2024-07

jackyliu16 reviewed (changes requested):
This update doesn't seem right. 

It's the first time that I have saw a update only update the version without hash at the same time. 

Did you build this package locally?
via nix-build . -A jetbrains-toolbox  or something else ?

and by the ways, using sha256 as param of fetch is not recommanded anymore
considerate to change it into hash and SRI hash (which could generate via `nix hash to-sri --type sha256 $(nix-prefetch-url <URL>)`.

```diff
-    sha256 = "sha256-5dgtaqi9cMpEoSl3MRaHWzCA8ktU5Mgjvt9CnBzr1ME=";
+   hash = "sha256- ... ";
```

jackyliu16 commented:
Also, divide one operation into two commit is not recommanded, you should use git rebase to squash it into one.

And Change the title of this PR and the commit message into a canonical naming, which should be `jetbrains-toolbox: 2.3.2.31487 -> 2.4.0.32175`

And I saw a kind of flags of first-time contribution, you should consider to add youself as maintainer (following the guideline in [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).)

Pandapip1 committed:
niri: add updateScript

sumnerevans committed:
matrix-synapse-unwrapped: 1.109.0 -> 1.110.0

https://github.com/element-hq/synapse/releases/tag/v1.110.0
Signed-off-by: Sumner Evans <me@sumnerevans.com>

wegank committed:
maven: 3.9.6 -> 3.9.8

wegank committed:
commafeed: update mvnHash

wegank committed:
global-platform-pro: update mvnHash

wegank committed:
gol: update mvnHash

wegank committed:
h2: update mvnHash

wegank committed:
jd-cli: update mvnHash

wegank committed:
kotlin-interactive-shell: update mvnHash

wegank committed:
slipstream: update mvnHash

wegank committed:
tabula-java: update mvnHash

wegank committed:
openrefine: update mvnHash

wegank committed:
runelite: update mvnHash

teto committed:
generateLuarocksConfig: dont hardcode local_cache

I tried using luaLib.generateLuarocksConfig outside nixpkgs, to generate a rocks.nvim config but it adds a `local_cache = ""` configuration line that is painful to remove so let's not add it in the first place, and let the caller set it if needed.

Bot-wxt1221 committed:
qq: 3.2.9 -> 3.2.10

pbsds committed:
dolphin-emu: switch to git release tags

pbsds committed:
dolphin-emu: refactor to finalAttrs pattern

hraban committed:
cloudflare-dyndns: configurable systemd frequency

bcdarwin committed:
python311Packages.monai: patch out distutils usage

Allows building the package with Python 3.12.

natsukium committed:
python311Packages.rubicon-objc: 0.4.8 -> 0.4.9

Diff: https://github.com/beeware/rubicon-objc/compare/refs/tags/v0.4.8...v0.4.9

Changelog: https://github.com/beeware/rubicon-objc/releases/tag/v0.4.9

r-ryantm committed:
komikku: 1.49.0 -> 1.50.0

Aleksanaa committed:
fetchNuGet: add SRI hash support

Aleksanaa committed:
mkNugetDeps: add SRI hash support

GaetanLepage committed:
cosmic-tasks: 0-unstable-2024-04-30 -> 0.1.0

Aleksanaa committed:
nuget-to-nix: generate SRI hash instead of sha256

Aleksanaa committed:
buildDotNetGlobalTool: add SRI hash support

Aleksanaa committed:
doc/dotnet: document SRI hash support in dotnet package

mweinelt committed:
matrix-appservice-irc: 2.0.0 -> 2.0.1

https://github.com/matrix-org/matrix-appservice-irc/releases/tag/2.0.1
https://github.com/matrix-org/matrix-appservice-irc/security/advisories/GHSA-w9mh-5x8j-9754

r-ryantm committed:
stress-ng: 0.18.00 -> 0.18.01

Aleksanaa committed:
treewide: change nugetSha256 to nugetHash

This is both to verify the correctness of the changes in dotnet helper
and to promote the use of nugetHash (which uses SRI hash).

For more information on nugetHash, check out
https://nixos.org/manual/nixpkgs/unstable/#packaging-dotnet-global-tools

Aleksanaa committed:
pablodraw: update deps.nix to use SRI hash

This is part of my test to prove that nuget-to-nix works with SRI hash.
It was built and ran successfully.

GaetanLepage committed:
cosmic-tasks -> tasks

GaetanLepage committed:
python311Packages.qcodes: 0.45.0 -> 0.46.0

Diff: https://github.com/microsoft/Qcodes/compare/refs/tags/v0.45.0...v0.46.0

Changelog: https://github.com/QCoDeS/Qcodes/releases/tag/v0.46.0

GaetanLepage committed:
python311Packages.cf-xarray: 0.9.1 -> 0.9.3

Diff:
https://github.com/xarray-contrib/cf-xarray/compare/refs/tags/v0.9.1...v0.9.3

Changelog:
https://github.com/xarray-contrib/cf-xarray/releases/tag/v0.9.3

GaetanLepage committed:
python311Packages.pytest-examples: 0.0.10 -> 0.0.12

Diff: https://github.com/pydantic/pytest-examples/compare/refs/tags/v0.0.10...v0.0.12

Changelog: https://github.com/pydantic/pytest-examples/releases/tag/v0.0.12

wegank committed:
vnote: move to pkgs/by-name

SuperSandro2000 committed:
gitea: 1.22.0 -> 1.22.1

Diff: https://github.com/go-gitea/gitea/compare/v1.22.0...v1.22.1

superherointj committed:
odin: refactor, enable aarch64-linux

dotlambda committed:
python312Packages.greeclimate: 1.4.6 -> 2.0.0

Diff: https://github.com/cmroche/greeclimate/compare/refs/tags/v1.4.6...v2.0.0

Changelog: https://github.com/cmroche/greeclimate/blob/refs/tags/v2.0.0/CHANGELOG.md

r-ryantm committed:
oxtools: 1.2.4 -> 2.0.2

redyf committed:
bruno: 1.20.0 -> 1.20.1

r-ryantm committed:
boogie: 3.1.6 -> 3.2.0

znaniye committed:
odin: nix-fmt format

r-ryantm committed:
ldc: 1.38.0 -> 1.39.0

r-ryantm committed:
wasmer: 4.3.2 -> 4.3.3

r-ryantm committed:
zx: 8.1.3 -> 8.1.4

NotAShelf committed:
nixos/roundcube: adjust maxAttachmentSize, clarify documentation

JohnRTitor committed:
rtl8821ce: move to finalAttrs, nixfmt-rfc-style

r-ryantm committed:
granted: 0.28.0 -> 0.29.0

justinas committed:
teleport_16: init at 16.0.4

drupol commented on pkgs/by-name/je/jetbrains-toolbox/package.nix:
```diff
@@ -9,11 +9,11 @@
   src = fetchzip {
     url = "https://download.jetbrains.com/toolbox/jetbrains-toolbox-${version}.tar.gz";
-    sha256 = "sha256-5dgtaqi9cMpEoSl3MRaHWzCA8ktU5Mgjvt9CnBzr1ME=";
+    sha256 = "sha256-+EKl8o+S5nwV3u+RrhxuOm/6dLH6zRSvFnyaZRO8xc0=";
```
```suggestion
    hash = "sha256-+EKl8o+S5nwV3u+RrhxuOm/6dLH6zRSvFnyaZRO8xc0=";
```

r-ryantm committed:
python311Packages.mkdocs-jupyter: 0.24.6 -> 0.24.8

znaniye committed:
pkgsMusl.odin: mark broken for musl

NickCao committed:
telegram-desktop: use finalAttrs pattern to simplify packaging tdesktop forks

znaniye committed:
odin: migrate to by-name

fabaff committed:
checkov: 3.2.171 -> 3.2.174

Diff: https://github.com/bridgecrewio/checkov/compare/refs/tags/3.2.171...3.2.174

Changelog: https://github.com/bridgecrewio/checkov/releases/tag/3.2.174

fabaff committed:
metasploit: 6.4.15 -> 6.4.16

fabaff committed:
python312Packages.boto3-stubs: 1.34.138 -> 1.34.139

fabaff committed:
python312Packages.botocore-stubs: 1.34.138 -> 1.34.139

fabaff committed:
python312Packages.pylacus: 1.9.0 -> 1.10.0

Diff: https://github.com/ail-project/PyLacus/compare/refs/tags/v1.9.0...v1.10.0

Changelog: https://github.com/ail-project/PyLacus/releases/tag/v1.10.0

r-ryantm committed:
melonDS: 0.9.5-unstable-2024-06-24 -> 0.9.5-unstable-2024-07-04

fabaff committed:
python312Packages.w3lib: 2.1.2 -> 2.2.1

Changelog: https://github.com/scrapy/w3lib/blob/v2.2.1/NEWS

r-ryantm committed:
wireless-regdb: 2024.05.08 -> 2024.07.04

fabaff committed:
python312Packages.playwrightcapture: 1.24.11 -> 1.25.0

Diff: https://github.com/Lookyloo/PlaywrightCapture/compare/refs/tags/v1.24.11...v1.25.0

Changelog: https://github.com/Lookyloo/PlaywrightCapture/releases/tag/v1.25.0

fabaff committed:
python311Packages.w3lib: refactor

fabaff committed:
python311Packages.parsel: refactor

- clean-up

fabaff committed:
python312Packages.easyenergy: 2.1.1 -> 2.1.2

Diff: https://github.com/klaasnicolaas/python-easyenergy/compare/refs/tags/v2.1.1...v2.1.2

Changelog: https://github.com/klaasnicolaas/python-easyenergy/releases/tag/v2.1.2

rhendric committed:
sudo: add rhendric as maintainer

fabaff committed:
python312Packages.archinfo: 9.2.108 -> 9.2.109

Diff: https://github.com/angr/archinfo/compare/refs/tags/v9.2.108...v9.2.109

fabaff committed:
python312Packages.ailment: 9.2.108 -> 9.2.109

Diff: https://github.com/angr/ailment/compare/refs/tags/v9.2.108...v9.2.109

fabaff committed:
python312Packages.pyvex: 9.2.108 -> 9.2.109

fabaff committed:
python312Packages.claripy: 9.2.108 -> 9.2.109

Diff: https://github.com/angr/claripy/compare/refs/tags/v9.2.108...v9.2.109

roberth committed:
nixosTests.docker-tools: fix for hello's new versionCheckHook

mweinelt committed:
evcc: 0.128.0 -> 0.128.1

https://github.com/evcc-io/evcc/releases/tag/0.128.1

fabaff committed:
python312Packages.cle: 9.2.108 -> 9.2.109

Diff: https://github.com/angr/cle/compare/refs/tags/v9.2.108...v9.2.109

fabaff committed:
python311Packages.angr: 9.2.108 -> 9.2.109

Diff: https://github.com/angr/angr/compare/refs/tags/v9.2.108...v9.2.109

fabaff committed:
python312Packages.lacuscore: 1.9.6 -> 1.10.0

Diff: https://github.com/ail-project/LacusCore/compare/refs/tags/v1.9.6...v1.10.0

Changelog: https://github.com/ail-project/LacusCore/releases/tag/v1.10.0

fabaff committed:
python312Packages.mkdocs-rss-plugin: 1.14.0 -> 1.15.0

Diff: https://github.com/Guts/mkdocs-rss-plugin/compare/refs/tags/1.14.0...1.15.0

Changelog: https://github.com/Guts/mkdocs-rss-plugin/blob/refs/tags/1.15.0/CHANGELOG.md

fabaff committed:
python312Packages.deebot-client: 8.0.0 -> 8.1.1

Diff: https://github.com/DeebotUniverse/client.py/compare/refs/tags/8.0.0...8.1.1

Changelog: https://github.com/DeebotUniverse/client.py/releases/tag/8.1.1

fabaff committed:
python312Packages.anova-wifi: 0.14.0 -> 0.15.0

Diff: https://github.com/Lash-L/anova_wifi/compare/refs/tags/v0.14.0...v0.15.0

Changelog: https://github.com/Lash-L/anova_wifi/releases/tag/v0.15.0

adamcstephens committed:
next-ls: init at 0.23.0

fabaff committed:
python312Packages.youless-api: 2.1.0 -> 2.1.2

Diff: https://github.com/gjong/youless-python-bridge/compare/refs/tags/2.1.0...2.1.2

Changelog: https://github.com/gjong/youless-python-bridge/releases/tag/2.1.2

fabaff committed:
python312Packages.inkbird-ble: 0.5.7 -> 0.5.8

Diff: https://github.com/Bluetooth-Devices/inkbird-ble/compare/refs/tags/v0.5.7...v0.5.8

Changelog: https://github.com/Bluetooth-Devices/inkbird-ble/blob/v0.5.8/CHANGELOG.md

Commit:
melonDS: darwin support

tadeokondrak committed:
treewide: remove maintainer tadeokondrak

SomeoneSerge committed:
release-cuda: import attributes from https://github.com/SomeoneSerge/nixpkgs-cuda-ci/

SomeoneSerge committed:
release-cuda: refine the allowUnfreePredicate

mweinelt committed:
esphome: 2024.6.4 -> 2024.6.6

https://github.com/esphome/esphome/releases/tag/2024.6.5
https://github.com/esphome/esphome/releases/tag/2024.6.6

imnotpoz committed:
anki-bin: inherit `mainProgram` from anki

olivia-fl committed:
maintainers: remove Benjamin-L

nope, absolutely not: <https://discourse.nixos.org/t/nca-member-jonringer-joint-announcement/48231>

Guanran928 committed:
swift: mark as broken on darwin (#324593)

azuwis committed:
moonlight-qt: upgrade to qt6

r-ryantm committed:
lunatask: 2.0.0 -> 2.0.4

r-ryantm committed:
freetds: 1.4.17 -> 1.4.19

r-ryantm committed:
debianutils: 5.19 -> 5.20

AndersonTorres committed:
melonDS: reword input set

r-ryantm committed:
cinnamon.mint-x-icons: 1.6.9 -> 1.7.0

r-ryantm committed:
simdjson: 3.9.4 -> 3.9.5

r-ryantm committed:
function-runner: 5.1.2 -> 5.1.3

r-ryantm committed:
typos: 1.22.9 -> 1.23.0

r-ryantm committed:
yanic: 1.5.2 -> 1.6.1

r-ryantm committed:
yamlscript: 0.1.63 -> 0.1.64

r-ryantm committed:
python311Packages.qcelemental: 0.27.1 -> 0.28.0

vbgl committed:
yices: 2.6.4 → 2.6.5

natsukium committed:
python311Packages.iodata: 1.0.0a2 -> 1.0.0a4

Diff: https://github.com/theochem/iodata/compare/refs/tags/v1.0.0a2...v1.0.0a4

natsukium committed:
python311Packages.iodata: enable check phase

r-ryantm committed:
mpvScripts.memo: 0-unstable-2023-10-31 -> 0-unstable-2024-07-02

r-ryantm committed:
omnictl: 0.38.2 -> 0.38.3

fabaff committed:
cnspec: 11.10.0 -> 11.11.0

Diff: https://github.com/mondoohq/cnspec/compare/refs/tags/v11.10.0...v11.11.0

Changelog: https://github.com/mondoohq/cnspec/releases/tag/v11.11.0

fabaff committed:
python312Packages.cyclonedx-python-lib: 7.4.1 -> 7.5.0

Diff: https://github.com/CycloneDX/cyclonedx-python-lib/compare/refs/tags/v7.4.1...v7.5.0

Changelog: https://github.com/CycloneDX/cyclonedx-python-lib/releases/tag/v7.5.0

fabaff committed:
python312Packages.cyclopts: 2.9.1 -> 2.9.3

Diff: https://github.com/BrianPugh/cyclopts/compare/refs/tags/v2.9.1...v2.9.3

Changelog: https://github.com/BrianPugh/cyclopts/releases/tag/v2.9.3

drupol reviewed (changes requested):
This PR should only contain 1 single commit.

fabaff committed:
python312Packages.find-libpython: 0.3.1 -> 0.4.0

Diff: https://github.com/ktbarrett/find_libpython/compare/refs/tags/v0.3.1...v0.4.0

Changelog: https://github.com/ktbarrett/find_libpython/releases/tag/v0.4.0

fabaff committed:
python312Packages.google-cloud-securitycenter: 1.31.0 -> 1.32.0

Changelog: https://github.com/googleapis/google-cloud-python/blob/google-cloud-securitycenter-v1.32.0/packages/google-cloud-securitycenter/CHANGELOG.md

fabaff committed:
python312Packages.hstspreload: 2024.6.1 -> 2024.7.1

Diff: https://github.com/sethmlarson/hstspreload/compare/refs/tags/2024.6.1...2024.7.1

GaetanLepage committed:
python311Packages.pint: 0.24 -> 0.24.1

Diff: https://github.com/hgrecco/pint/compare/refs/tags/0.24...0.24.1

Changelog: https://github.com/hgrecco/pint/blob/0.24.1/CHANGES

r-ryantm committed:
home-manager: 0-unstable-2024-06-26 -> 0-unstable-2024-07-05

r-ryantm committed:
rs-tftpd: 0.2.13 -> 0.3.0

r-ryantm committed:
python311Packages.oelint-parser: 3.5.3 -> 3.5.4

Sigmanificient committed:
wakatime-cli: 1.90.0 -> 1.93.0

GaetanLepage committed:
python311Packages.pint-pandas: 0.4 -> 0.6

Diff: https://github.com/hgrecco/pint-pandas/compare/0.4...0.6

r-ryantm committed:
iosevka-bin: 30.3.0 -> 30.3.1

Vinetos committed:
nova-password: init at 0.5.6

GaetanLepage committed:
nbqa: disable failing tests

Commit:
pulumi-bin: 3.121.0 -> 3.122.0

GaetanLepage committed:
oboete: 0.1.2 -> 0.1.3

Diff: https://github.com/mariinkys/oboete/compare/refs/tags/0.1.2...0.1.3

Changelog: https://github.com/mariinkys/oboete/releases/tag/0.1.3

shivaraj-bh committed:
python3Packages.pyreqwest-impersonate: 0.4.7 -> 0.4.9

shivaraj-bh committed:
python3Packages.duckduckgo-search: 5.3.1 -> 6.1.9

r-ryantm committed:
mame: 0.266 -> 0.267

alyssais committed:
libeatmydata: patch out LFS64 usage (#324772)

This is a revised version of c7f51f003264 ("libeatmydata: patch out
LFS64 usage") (reverted in 7be23ec058c8 ("Revert "libeatmydata: patch
out LFS64 usage"")) that doesn't break 32-bit Glibc builds.

r-ryantm committed:
mpc-qt: 23.12 -> 24.06

AndersonTorres committed:
xonsh: remove AndersonTorres from meta.maintainers

Since I can't keep this.

amiryal committed:
jujutsu: 0.18.0 -> 0.19.0

AndersonTorres committed:
melonDS: use qt6 libraries

r-ryantm committed:
xcp: 0.21.1 -> 0.21.2

r-ryantm committed:
python311Packages.amberelectric: 1.1.0 -> 1.1.1

r-ryantm committed:
xed-editor: 3.6.3 -> 3.6.4

r-ryantm committed:
cargo-tally: 1.0.47 -> 1.0.48

r-ryantm committed:
folio: 24.10 -> 24.11

r-ryantm committed:
git-toolbelt: 1.9.2 -> 1.9.3

r-ryantm committed:
gqlgenc: 0.22.0 -> 0.23.2

K900 committed:
linux_testing: 6.10-rc5 -> 6.10-rc6

K900 committed:
linux_6_9: 6.9.7 -> 6.9.8

K900 committed:
linux_6_6: 6.6.36 -> 6.6.37

K900 committed:
linux_6_1: 6.1.96 -> 6.1.97

K900 committed:
linux_5_15: 5.15.161 -> 5.15.162

K900 committed:
linux_5_10: 5.10.220 -> 5.10.221

K900 committed:
linux_5_4: 5.4.278 -> 5.4.279

K900 committed:
linux_4_19: 4.19.316 -> 4.19.317

K900 committed:
linux-rt_5_10: 5.10.218-rt110 -> 5.10.219-rt111

K900 committed:
linux-rt_5_4: 5.4.271-rt89 -> 5.4.278-rt91

K900 committed:
linux-rt_6_1: 6.1.95-rt34 -> 6.1.96-rt35

K900 committed:
linux-rt_6_6: 6.6.35-rt34 -> 6.6.36-rt35

r-ryantm committed:
mtail: 3.0.3 -> 3.0.5

r-ryantm committed:
oven-media-engine: 0.16.5 -> 0.16.6

r-ryantm committed:
automatic-timezoned: 2.0.18 -> 2.0.21

GaetanLepage committed:
pylyzer: 0.0.54 -> 0.0.55

Diff: https://github.com/mtshiba/pylyzer/compare/refs/tags/v0.0.54...v0.0.55

Changelog: https://github.com/mtshiba/pylyzer/releases/tag/v0.0.55

4evy committed:
raycast: 1.78.0 -> 1.78.1

4evy committed:
arc-browser: 1.49.0-51346 -> 1.49.1-51495

r-ryantm committed:
ananicy-rules-cachyos: 0-unstable-2024-06-22 -> 0-unstable-2024-07-03

r-ryantm committed:
prometheus-sql-exporter: 0.5.4 -> 0.5.5

r-ryantm committed:
xarcan: 0-unstable-2024-05-11 -> 0-unstable-2024-05-23

jansol committed:
zed-editor: fix .desktop file

r-ryantm committed:
vdrPlugins.softhddevice: 2.3.3 -> 2.3.4

r-ryantm committed:
piv-agent: 0.21.1 -> 0.21.2

r-ryantm committed:
bitmagnet: 0.9.3 -> 0.9.4

r-ryantm committed:
dasel: 2.8.0 -> 2.8.1

r-ryantm committed:
oh-my-zsh: 2024-06-28 -> 2024-07-03

flokli committed:
buildRustCrate: don't swallow build.rs output

This is executed curing the build, and usually quite helpful in telling
you how the build failed, if it failed.

For example, when linking against system libraries, it shows which
system lib could not be discovered, the pkg-config path used etc.

Not showing it simply removes all this, and trying to parse out the
"important" bits and only show these is probably also brittle.

The output generally isn't that spammy normally, so let's just `tee` it.

drupol commented:
It looks like you accidentally mass-pinged a bunch of people, which are now subscribed
and getting notifications for everything in this pull request. Unfortunately, they
cannot be automatically unsubscribed from the issue (removing review request does not
unsubscribe), therefore development cannot continue in this pull request anymore.

Please create a new pull request, link back to this one and ping the
people actually involved in here over there. For the next time, remember to set your PR to draft status before rebasing. In draft status, you can preview the list of maintainers that are about to be requested for review, which allows you to sidestep this issue.

In order to avoid this in the future, there are instructions for how to properly
rebase between branches in our [contribution guidelines](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#rebasing-between-branches-ie-from-master-to-staging).
Setting your pull request to draft prior to rebasing is strongly recommended.
In draft status, you can preview the list of people that are about to be requested
for review, which allows you to sidestep this issue.
This is not a bulletproof method, though, as OfBorg still does review requests even on draft PRs.
